components/homepage/sections/key-features/Layout.tsx

```diff
@@ -0,0 +1,46 @@
+import { Flex } from "@chakra-ui/react";
+import React from "react";
+import { Heading, HeadingProps } from "tw-components";
+
+type LayoutProps = {
+  title: string;
+  titleGradient: HeadingProps["bgGradient"];
+  headline: string;
+  description: string;
+  children: React.ReactNode;
+};
+export const Layout: React.FC<LayoutProps> = ({
```

Comment:
```suggestion
export const Layout: ComponentWithChildren<LayoutProps> = ({
```

please use our ComponentWithChildren custom type so you don't have to add children to the props

Also not sure I like "Layout" as this component name. can we change it so something like "KeyFeaturesLayout"?